src/interfaces/IERC5827Spender.sol

```diff
@@ -3,6 +3,7 @@ pragma solidity 0.8.17;
 
 /// @title IERC5827Spender defines a callback function that is called when renewable allowance is approved
 /// @author Zlace
+/// @notice This interface must be imported if the contract wants to spend renewable allowance
```

Comment:
this isn't true. Only true if spender wants to be informed of a renewable approval 
```suggestion
/// @notice This interface must be implemented if the spender contract wants to react to an renewable approval
```